doc/funding/priorities.md

```diff
@@ -0,0 +1,68 @@
+
+### Key outcomes
+
+1. **Increase contributor perception of the software quality.** Jupyter's community members are valid assessors of Jupyter’s software quality, since many are professional software engineers and because their perception influences their engagement with Jupyter. We will measure their perception of Jupyter’s software quality through community outreach efforts, such as a semi-annual survey.
+2. **Increase testing code coverage.** Software testing is a fundamental aspect of ensuring a base level of software quality. We will identify a set of key repositories based on usage and dependencies and track the amount of source code covered by tests at all levels.  
+3. **Increase the frequency and transparency of minor releases.** “Release early, release often”, with an emphasis on minor releases providing bug fixes, security patches, non-disruptive performance improvements, and similarly non-disruptive new features. Ensure that releases always make it clear what has changed and what it means for a user. Minor releases with these characteristics are easier for Jupyter users and operators to deploy. This pattern also keeps the contributors familiar with the release process.  
+4. **Increase the speed of security fixes.** Security vulnerabilities are a strong concern for organizations that deploy Jupyter technology as part of shared services. Improvements to Jupyter’s overall software quality should result in faster, more efficient processes for identifying, resolving, and releasing security errors.
+
+### Examples of how to grow software quality
+
+- Hold events that allow subprojects to meet and co-work to solve their most pressing problems.  
+- Fund contributor time to focus on security, testing coverage, etc.  
+- Fund contributor time to create more consistent releases with higher-quality release notes across the project.  
+- Fund development efforts to build common infrastructure for releases, documentation, and testing across the Jupyter ecosystem.  
+- Fund contributor time to focus on stewarding pull-requests, with a focus on maintenance and bug fixes.  
+- Fund contributor time for stewarding JEPs and encouraging productive and efficient cross-community conversation.
+
+## Ensure that Jupyter has a reliable source of funding by developing the Jupyter Foundation 
+
+The funding mechanisms we define will only be effective if we can reliably engage and grow the members of the Jupyter Foundation. Doing so will require dedicated capacity to ensure that the Foundation is delivering value to its members, and to engage *prospective* members to grow the foundation.
+
+By using Foundation funding to **develop the Foundation membership and ensure it delivers value to them**, we will increase the likelihood that our foundation’s members are engaged and satisfied with their participation. This will reduce churn and increase our chances of increasing financial resources for the project.
```

Comment:
```suggestion
By using Foundation funding to **develop the Foundation membership and ensure it delivers value to them**, we will increase the likelihood that the Foundation’s members are engaged and satisfied with their participation. This will reduce churn and increase our chances of increasing financial resources for the project.
```